kitsune/settings.py

```diff
@@ -537,6 +537,9 @@ def parse_conn_max_age(value):
     'commonware.middleware.RobotsTagHeader',
     # 'axes.middleware.FailedLoginMiddleware'
     'django_user_agents.middleware.UserAgentMiddleware',
+
+    'kitsune.users.middleware.LogoutDeactivatedUsersMiddleware',
```

Comment:
We still have `ModelBackendAllowInactive` enabled:
https://github.com/akatsoulas/kitsune/blob/61c21004263ef7575c9a6aabe67e974a268d1f45/kitsune/settings.py#L572